examples/gadgets/parser/trace/exec/exec.go

```diff
@@ -90,20 +89,20 @@ func main() {
 	}
 	defer containerCollection.Close()
 
-	// Create a parser. It's the component that converts events to columns.
-	execParser, err := utils.NewGadgetParser(&utils.OutputConfig{}, types.GetColumns())
-	if err != nil {
-		fmt.Printf("failed to create parser: %s\n", err)
-		return
-	}
+	// Create a formatter. It's the component that converts events to columns.
+	colNames := []string{"container", "pid", "ppid", "pcomm", "ret", "args"}
```

Comment:
`pcomm` -> `comm`. In fact, it is missing in the output.

Reply:
We need to fix it in other places as well. The member name is comm but it's pcomm in json:

https://github.com/kinvolk/inspektor-gadget/blob/2bd017d5915820529ca34480fbdba5fd44b372d4/pkg/gadgets/trace/exec/types/types.go#L29

And we also have PCOMM on our guides -> https://github.com/kinvolk/inspektor-gadget/blob/2bd017d5915820529ca34480fbdba5fd44b372d4/docs/guides/trace/exec.md :(